src/meta.ts

```diff
@@ -16,6 +18,12 @@ export default (pwa: PWAContext) => {
   // mobileApp (IOS)
   if (options.mobileAppIOS) {
     head.meta.push({ name: 'apple-mobile-web-app-capable', content: 'yes' })
+    for (const device of devices) {
+      const href = join(pwa._assetsDir, pwa.icon.targetDir, `${device.width}x${device.height}-splash-screen.png`)
```

Comment:
It should be `nuxt.options.app.buildAssetsDir` instead of `pwa._assetsDir`
> See : https://github.com/kevinmarrec/nuxt-pwa-module/blob/main/src/icon.ts#L42